src/legacy/core_plugins/metrics/public/components/aggs/moving_average.js

```diff
@@ -34,58 +33,82 @@ import {
   EuiComboBox,
   EuiSpacer,
   EuiFormRow,
-  EuiCode,
-  EuiTextArea,
+  EuiFieldNumber,
 } from '@elastic/eui';
-import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import { MODEL_TYPES } from '../../../common/model_options';
 
-const MovingAverageAggUi = props => {
-  const { siblings, intl } = props;
-  const defaults = {
-    settings: '',
-    minimize: 0,
-    window: '',
-    model: 'simple'
-  };
-  const model = { ...defaults, ...props.model };
-  const handleChange = createChangeHandler(props.onChange, model);
-  const handleSelectChange = createSelectHandler(handleChange);
-  const handleTextChange = createTextHandler(handleChange);
-  const handleNumberChange = createNumberHandler(handleChange);
+const DEFAULTS = {
+  model_type: MODEL_TYPES.UNWEIGHTED,
+  alpha: 0,
+  beta: 0,
+  gamma: 0,
+  period: 1,
+  multiplicative: true,
+  window: 10,
```

Comment:
I think we should check those values against the existing implementation to find the right default one.
For example, default `''` window with the old moving avg seems to be equal to a window size of 5 (is the default value from the docs: https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-pipeline-movavg-aggregation.html#_syntax_8)

I've also found that an `exponential weighted` with alpha 0.3 is equal to the old exponentional weighted with no minimize and no predict.

A holt linear with alpha 0.3 and beta 0.1 seems equal to an old holt linear.

I didn't find a good value for the holt winters.

Please take a 7.1 instance of ES+Kibana, reindex the same data on this cluster with: https://www.elastic.co/guide/en/elasticsearch/reference/current/reindex-upgrade-remote.html
to have the same data and try to find a good defaults that can be considered as much as possible similar to the previous one.
Try also to add the same values on the migrations

Reply:
Nice catch! Proper default values are set. According to the documentation default values are equal regardless a model type. And we've got essentials for the `holt-winter` type: alpha, beta, gamma, period, window.